c/cryptography/cryptography_ubi_9.3.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package           : cryptography
+# Version           : 38.0.1
+# Source repo       : https://github.com/pyca/cryptography.git
+# Tested on         : UBI:9.3
+# Language          : Python
+# Travis-Check      : True
+# Script License    : Apache License, Version 2.0
+# Maintainer        : Md. Shafi Hussain <Md.Shafi.Hussain@ibm.com>
+#
+# Disclaimer        : This script has been tested in root mode on given
+# ==========          platform using the mentioned version of the package.
+#                     It may not work as expected with newer versions of the
+#                     package and/or distribution. In such case, please
+#                     contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=cryptography
+PACKAGE_VERSION=${1:-38.0.1}
+PACKAGE_URL=https://github.com/pyca/cryptography.git
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm \
+    git redhat-rpm-config gcc libffi-devel python3-devel openssl-devel cargo pkg-config
+
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+pip3 install wheel setuptools_rust
```

Comment:
Use just pip and python instead of pip3 and python3

Reply:
Done